### DISCUSSION TITLE
error in saving from eden

### DISCUSSION THREAD

**Discussion by nikodiseas** [jetelain/ArmaRealMap#103] [General]
Hi. I'm trying to save the changes from the EDEN editor to the mod, but for some reason I can't: the program freezes on binarization of the WRP file. My map is the size of an Altis, however, I decided to fill it out slowly. But could it depend on the size? Could it be related to Altis/Stratis textures? What do you say? Here is my error file. As you can see, something can't load there...
[studio_2023-08-23.txt](https://github.com/jetelain/ArmaRealMap/files/12414060/studio_2023-08-23.txt)

**Comment by nikodiseas** [comment]
By the way, I would like to say a few more words about rivers. The fact is that there is never water in them, whereas simple reservoirs contain water.

**Comment by nikodiseas** [comment]
I will also add to the main problem that if you restart the program and open your WRP file, the name of the map (config description) will be missing, and the "generate mod" button will stop working. Only rolling back to the original version helps. I don't understand why this is so. Either the size, or the texture, or, I forgot to add, a large number of edits (16 thousand objects).

**Comment by jetelain** [comment]
Hi,
I already did edit of thousand of objects without issue. According to the logs, it's the binarization that takes times. You may try to binarize with an other tool like PboProject. On very large maps, binarization can take up to one hour, depending on tree/rocks count.
The binarization can freeze the UI if the "output" is open when it writes too much messages. It's something I have to fix in an upstream dependency (gemini).
The name of the map is read from the config.cpp, if the name is missing, something might get wrong in the file. May you send me the config.cpp when the issue appears ?
